src/model/page.urls.ts

```diff
@@ -5,7 +5,11 @@ import { Templates } from "./template.paths";
  */
 
 const SEPARATOR: string = "/";
+export const OBJECTIONS: string = SEPARATOR + "strike-off-objections";
 
 export const STRIKE_OFF_OBJECTIONS: string = SEPARATOR + "strike-off-objections";
 export const COMPANY_NUMBER: string = SEPARATOR + Templates.COMPANY_NUMBER;
 export const CONFIRM_COMPANY: string = SEPARATOR + Templates.CONFIRM_COMPANY;
+
+export const OBJECTIONS_COMPANY_NUMBER: string = OBJECTIONS + COMPANY_NUMBER;
```

Comment:
A bit confused why we need these as well as the two definitions above. These two have the full path the ones above don't.

Reply:
Ah just seen the same was done in PTF so probably needed (for redirects). Worth lifting the comments in the PTF page.urls file and copying here I think - they explain why there is partial duplication...

Reply:
We don't always need the full path but sometimes we do.